engine/render/src/dmsdk/render/render.h

```diff
@@ -262,6 +269,43 @@ namespace dmRender
         RENDER_ORDER_AFTER_WORLD  = 2,
     };
 
+    /*#
+     * Visibility status
+     * @enum
+     * @name Visibility
+     * @member VISIBILITY_NONE
+     * @member VISIBILITY_FULL
+     */
+    enum Visibility
+    {
+        VISIBILITY_NONE = 0,
+        VISIBILITY_FULL = 1, // Not sure if we ever need partial ?
```

Comment:
Perhaps if an object is covering a very small area of the screen, or if the bounding volume is super small we maybe want to skip rendering it eventhough its technically visible? Some kind of configurable parameter for visibility barrier or something to set the limits when this kicks in. Or maybe it's more related to a LOD system 🤔

Reply:
Yeah, I'd put that into the lod system